jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -29,10 +29,16 @@ pipeline {
     environment {
         BUILD_MANIFEST = "build-manifest.yml"
         BUILD_MANIFEST_OPENSEARCH = "build-manifest-opensearch.yml"
-        BUILD_JOB_NAME = "distribution-build-opensearch-dashboards"
-        BUILD_JOB_NAME_OPENSEARCH = "distribution-build-opensearch"
+        BUILD_JOB_NAME = "distribution-build-opensearch-dashboards-dashboards"
+        BUILD_JOB_NAME_OPENSEARCH = "distribution-build-opensearch-dashboards"
```

Comment:
Why adding dashboards after the dashboards?

Reply:
Bad replacement. Fixed it now.